build-system/conformance-config.textproto

```diff
@@ -11,56 +11,58 @@ requirement: {
 
 requirement: {
   type: BANNED_PROPERTY_CALL
-  error_message: 'String.prototype.padStart is not allowed'
-  value: 'String.prototype.padStart'
+  error_message: 'string.prototype.padStart is not allowed'
```

Comment:
This is a bit of an ambiguous case. The way to get to the actual function in JS is to use `String.prototype.padStart`, but the name of the unboxed type is `string`